server/routes/induction/update/personalInterestsUpdateController.ts

```diff
@@ -0,0 +1,26 @@
+import { NextFunction, Request, RequestHandler, Response } from 'express'
+import PersonalInterestsController from '../common/personalInterestsController'
+
+/**
+ * Controller for the Update of the Personal Interests screen of the Induction.
+ */
+export default class PersonalInterestsUpdateController extends PersonalInterestsController {
+  constructor() {
+    super()
+  }
+
+  getBackLinkUrl(req: Request): string {
+    const { prisonNumber } = req.params
+    return `/plan/${prisonNumber}/view/work-and-interests`
+  }
+
+  getBackLinkAriaText(_req: Request): string {
+    return 'Back to <TODO - check what CIAG UI does here>'
```

Comment:
I'll look to resolve this TODO (and the other one below) shortly; this PR is intended to be just enough to get Adam moving 👍